Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/README.md

```diff
@@ -1,9 +1,9 @@
 Automate your AppID Adoption by using this integration together with your Palo Alto Networks Next-Generation Firewall or Panorama.
 This integration was integrated and tested with version 8 up to 10.1.6 of PAN-OS Policy Optimizer, later versions may not work
```

Comment:
Are you sure about the versions?

Reply:
I made sure with Gal (we had a meeting last Thursday) it was the previous version it was tested on.
I'm waiting for a new instance (https://jira-hq.paloaltonetworks.local/browse/CRTX-67271 ) to check it on version 10.1.12